components/command/command-handler/pom.xml

```diff
@@ -12,11 +12,6 @@
     <artifactId>command-handler</artifactId>
 
     <dependencies>
-        <dependency>
-            <groupId>javax</groupId>
-            <artifactId>javaee-api</artifactId>
-            <scope>provided</scope>
-        </dependency>
```

Comment:
We had some class path issues during maven build hence moved it down which seems to solve it at that time, but does not seem to appear any more so moved back to where it was